pallets/committee/src/benchmarking.rs

```diff
@@ -0,0 +1,216 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+use frame_benchmarking::benchmarks_instance;
+use frame_support::{dispatch::DispatchResult, traits::UnfilteredDispatchable, StorageValue};
+use polymesh_common_utilities::{
+    benchs::{User, UserBuilder},
+    MaybeBlock,
+};
+use sp_std::prelude::*;
+
+const COMMITTEE_MEMBERS_NUM: usize = 10;
+const COMMITTEE_MEMBERS_MAX: u32 = 100;
+const PROPOSAL_PADDING_LEN: usize = 10_000;
+const PROPOSALS_NUM: u8 = 100;
+
+fn make_proposals_and_vote<T, I>(users: &[User<T>]) -> DispatchResult
+where
+    I: Instance,
+    T: Trait<I>,
+{
+    for i in 0..PROPOSALS_NUM {
+        let index = Module::<T, I>::proposal_count();
```

Comment:
Could read this once and then just increment a counter instead.
Efficiency isn't crucial here, but might as well...

Reply:
I placed it within the loop in order not to diverge the index computation from the actual storage value. Efficiency isn't important during the setup phase.